src/org/labkey/test/util/DataRegionTable.java

```diff
@@ -137,6 +138,15 @@ public PagingWidget getPagingWidget()
         return _pagingWidget;
     }
 
+    public SetDefaultPageClass clicksetDefault()
+    {
+        if (!getCustomizeView().isPanelExpanded())
```

Comment:
I'm curious why this if statement is needed. Does the "Set Default" option not work or show for some reason if the custom view grid is expanded?